build.gradle.kts

```diff
@@ -50,3 +77,83 @@ subprojects {
         }
     }
 }
+
+/**
+ * To run detekt simply:
+ * 1. ./gradlew module:detekt for each module
+ * 2. ./gradlew detekt for whole project
+ */
+fun Project.coreDetektSetup() {
+    // Apply Plugin to sub-project
```

Comment:
this would be good to also add to the README for the time being. Probably we can add sections for CI/CD and this can go under that or under it's own section. Eventually we want this to run before merging, whether that is from commit hooks or jobs run through github actions is still something to work through

- [ ] Update README

Reply:
I'll add that in the README